src/js/components/Heading/doc.js

```diff
@@ -17,7 +17,7 @@ export const doc = (Heading) => {
       `A color identifier to use for the text color. For example:
       'brand'.`
     ),
-    level: PropTypes.oneOf([1, 2, 3, 4, '1', '2', '3', '4']).description(
+    level: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).description(
```

Comment:
So, this will not complain if someone tried "h1", but that isn't going to work. Any reason not to just extend to include 1-6 individually?

Reply:
yeah I think what I did was messy. i agree with you, let me change it.